docker-for-ibm-cloud/administering-swarms.md

```diff
@@ -78,16 +79,17 @@ To create a Docker EE for IBM Cloud cluster from the CLI:
    > export D4IC_DOCKER_EE_URL=my_docker-ee-url
 
 
-5. Note the cluster **Name** and **ID**.
+5. Note the cluster **Name**, **ID**, and **UCP Password**.
 
    > Swarm provisioning
    >
    > Your cluster is provisioned in two stages, and takes a few minutes to provision. Don't try to modify your cluster just yet!
    > First, the manager node is deployed. Then, the additional infrastructure resources are deployed, including the worker nodes, DTR nodes, load balancers, subnet, and NFS volume.
 
    * **Provisioning Stage 1**: Check the status of the manager node:
+
      {% raw %}
-    ```bash
+     ```bash
```

Comment:
This is incorrect. Everything should line up with the start of the text inside the bullet (so the double asterisk). It may or may not render correctly with incorrect indentation, so best not to risk it.